File: Curso-de-POO/Java/Account.java
class Account {
    Integer id;
    String name;
    String document;
    String email;
    String password;

    public Account(String name, String document, String email, String password){
        this.name = name;
        this.document = document;
        this.email = email;
        this.password = password;
    }
}

class User extends Account {
    public User(String name, String document, String email, String password){
        super(name, document, email, password);
    }
}

class Driver extends Account {
    public Driver(String name, String document, String email, String password){
        super(name, document, email, password);
    }
}

File: Curso-de-POO/JS/Account.js
class Account{    
    constructor(name, document, email, password){
        this.id;
        this.name = name;
        this.document = document;
        this.email = email;
        this.password = password;
    }
}

class User extends Account{
    constructor(name, document, email, password){
        super(name, document, email, password)
    }
}

class Driver extends Account{
    constructor(name, document, email, password){
        super(name, document, email, password)
    }
}